pages/send/Send.tsx

```diff
@@ -23,10 +23,22 @@ export function Send() {
   const [isLoading, setLoading] = React.useState(false);
   const [keyboardOpen, setKeyboardOpen] = React.useState(false);
   const [keyboardText, setKeyboardText] = React.useState("");
+  const [startScanning, setStartScanning] = React.useState(false);
 
+  // Delay starting the QR scanner if url has no valid payment info
   useEffect(() => {
     if (url) {
-      loadPayment(url);
+      (async () => {
+        try {
+          const result = await loadPayment(url);
+          setStartScanning(!result);
```

Comment:
maybe this could be simplified by returning if the result is ok, and always start scanning at the bottom of the function. Then there is no need for if/else.

Reply:
Just tried, but I think that makes it harder to read / understand.